storage_controller/migrations/2024-07-26-140924_create_leader/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE leader (
```

Comment:
nit: we could call this something more general like `controllers` in case we later decide to have some warm standbys: we'd probably want to track the non-leaders in the same table.

Reply:
Good idea: https://github.com/neondatabase/neon/pull/8588/commits/79a53f92c9fec888a1f2f71368fca48ec844a6c2